api/features/views.py

```diff
@@ -110,10 +111,19 @@ def perform_update(self, serializer):
         serializer.save(project_id=self.kwargs.get("project_pk"))
 
     def perform_destroy(self, instance):
-        self._create_feature_delete_audit_log(instance)
+        feature_states = instance.feature_states.filter(
+            identity=None, feature_segment=None
+        )
+        self._create_feature_delete_audit_log(feature_states)
+        self._trigger_feature_state_change_webhooks(feature_states)
         instance.delete()
 
-    def _create_feature_delete_audit_log(self, feature: Feature):
+    def _trigger_feature_state_change_webhooks(self, feature_states: FeatureState):
```

Comment:
I think the type hint is wrong here? `typing.Iterable[FeatureState]` or `QuerySet[FeatureState]` perhaps (not sure if that last one is valid syntax or not...?